src/components/Main.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Animals from './Animals';
+import Images from './Images';
+
+import styles from './Main.css';
+
+class Main extends React.PureComponent {
+    constructor() {
+        super();
+        this.state = {
+            animals: [],
+        };
+    };
+
+    fetchAnimals () {
+        const url = "http://localhost:3000/animals?amount=5"
+        return fetch(url,{
+            method: 'get',
+        })
+        .then(response => response.json())
+        .catch(error => {
+            console.log("Incorrect data from server", error)
+        });
+    }
+    componentDidMount() {
+        this.fetchAnimals().then(data => {
+            console.log("state", data)
+            this.setState({ animals: data.animals })
+        });
+    }
+
+    render() {
+        return (
+            <div className={styles.mainPage}>
+                <div className="container">
```

Comment:
You shouldn't have mix both css modules approach and classic className's in a file.

Reply:
Good point, I deleted this div container.